pkg/api/repository_rpm.go

```diff
@@ -29,6 +29,16 @@ type RepositoryRpmCollectionResponse struct {
 	Links Links `json:"links"`
 }
 
+type SearchRpmRequest struct {
+	URLs   []string `json:"urls"`
+	Search string   `json:"search"`
+}
+
+type SearchRpmResponse struct {
+	PackageName string `json:"package_name" gorm:"column:package_name"`
+	Summary     string `json:"summary" gorm:"column:summary"`
```

Comment:
just curious, does specifying the column names here had functionality somewhere?

Reply:
Honestly I don't remember the reason because I included it; my current understanding is that `gorm` will map the to the `snake_case` by convention, so it is not necessary to specify because the convention match the specified column name. Thanks for the heads up ;)  I will remove it.

Reply:
wait!!! It is not into the `pkg/models` side; it does not make sense at all the annotation; I have removed it. thanks